packages/main/src/Option.js

```diff
@@ -66,6 +66,16 @@ const metadata = {
 		stableDomRef: {
 			type: String,
 		},
+
+		/**
+		 * Defines the focused state of the <code>ui5-option</code>.
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @private
```

Comment:
As it is a private property, you can rename it to ```_focused```. Moreover, you can add ```@since 1.0.0-rc.13``` in the description of the property.